app/css/question.css

```diff
@@ -1,12 +1,35 @@
-#question_field {
+.QuestionForm {
+    border-top: 1px solid #e7e7e7;
+}
+.QuestionField {
     border: 0;
-    max-height: 100px;
+    color: #000;
+    font-size: 17px;
+    max-height: 142px;
     overflow: auto;
-    padding: 0.5em 1em;
+    padding: 10px 15px;
     outline: none;
     resize: none;
     width: 100%;
 }
+.QuestionField::-moz-placeholder {
+    color: #858585;
+    font-style: italic;
+}
+.QuestionSubmitButton {
```

Comment:
Instead of overriding all the building blocks, I think we can use `<input type=submit>`.